CHANGELOG.md

```diff
@@ -30,8 +30,10 @@ scheme are considered to be bugs.
 
 * [#435](https://github.com/intridea/hashie/pull/435): Mash `default_proc`s are now propagated down to nested sub-Hashes - [@michaelherold](https://github.com/michaelherold).
 * [#436](https://github.com/intridea/hashie/pull/436): Ensure that `Hashie::Extensions::IndifferentAccess` injects itself after a non-destructive merge - [@michaelherold](https://github.com/michaelherold).
+* [#438](https://github.com/intridea/hashie/pull/438): Require `pathname` stdlib that required in `Hashie::Mash.load` - [@onk](https://github.com/onk).
```

Comment:
You should list the problem here, so "Fix: NameError (uninitialized constant Hashie::Extensions::Parsers::YamlErbParser::Pathname) in Mashie::Mash.load" instead of saying what the fix was.